learning_journal_basic/learning_journal_basic/requirements.txt

```diff
@@ -0,0 +1,83 @@
+ackermann==0.1
```

Comment:
Is this actually used in your learning journal?  Check your requirements for unnecessary packages.